src/jit/target.h

```diff
@@ -484,9 +484,15 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
   #define MIN_ARG_AREA_FOR_CALL    0       // Minimum required outgoing argument space for a call.
 
   #define CODE_ALIGN               1       // code alignment requirement
+#if !defined(FEATURE_PAL)
   #define STACK_ALIGN              4       // stack alignment requirement
```

Comment:
This 16-byte stack alignment requirement comes from GCC/Clang (as discussed in#8590). I'm not sure which flag will be better: ``__GNUC__`` or ``FEATURE_PAL``.